src/index.ts

```diff
@@ -4,6 +4,7 @@
 /* ================================================================================================================== *
  * Authors:                                                                                                           *
  *   Unai Martinez-Corral                                                                                             *
```

Comment:
This guy's code might no longer be present in the current revision?

Reply:
Yeah. Should I remove the whole header or just the name?